frontend/src/modules/schedule-maker/contexts/TournamentScheduleContext.js

```diff
@@ -2,14 +2,24 @@ import { createContext, useMemo } from 'react';
 import { blocks, days, sports, areas } from '../utils/mocks';
 import { propEq } from 'ramda';
 import PropTypes from 'prop-types';
+import { useParams } from 'react-router-dom';
 
 export const tournamentScheduleContext = createContext(null);
 
 const TournamentScheduleProvider = tournamentScheduleContext.Provider;
 
 const TournamentScheduleContext = ({ children, tournament }) => {
-  //TODO: useParams
-  const detailMode = false;
+  //start region to by chtělo asi refaktor
+  const { detailModeParam } = useParams();
```

Comment:
Ano je to prasácký :D koukni na useUrlSearchParams https://reactrouter.com/en/v6.3.0/getting-started/tutorial#search-params

Reply:
udělal jsem to s tím searchparams, ale ta moje logika je docela dummy xd

Reply:
done, s tím usesearchparams je to lepší